tests/common/plugins/log_section_start/__init__.py

```diff
@@ -28,6 +32,78 @@ def pytest_configure(config):
     config.pluginmanager.register(LogSectionStartPlugin(logging_plugin), "LogSectionStart")
     logging.LogRecord = _LogRecord
 
+    postimport.register_hook(_pytest_import_callback)
+    # reload to inject custom pytest.fixture decorator
+    reload(pytest)
```

Comment:
Reload the whole `pytest`? Could this cause side effect?

Reply:
Yes, will update to simply replace the original `fixture`.